FirebaseAdmin/FirebaseAdmin/Auth/Providers/OidcProviderConfig.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Newtonsoft.Json;
+
+namespace FirebaseAdmin.Auth.Providers
+{
+    /// <summary>
+    /// Represents an OIDC auth provider configuration. See
+    /// https://openid.net/specs/openid-connect-core-1_0-final.html.
```

Comment:
Maybe having a hyperlink would be nice here

Reply:
Done